README.md

```diff
@@ -45,6 +45,7 @@ A list of currently supported fields
 - `origin` Source repository to mirror from
 - `skip`   Temporarily exclude a project from syncing by adding `skip: true`
 - `destination` Reserved for future use
+- `branch` Clone only single branch. Add `branch: <branchname>` to use it
```

Comment:
I think this should be called `refspec`. I think all of  https://git-scm.com/book/en/v2/Git-Internals-The-Refspec can be used here?

Reply:
Done